### DISCUSSION TITLE
The AI response is very short

### DISCUSSION THREAD

**Discussion by emimix** [ILikeAI/AlwaysReddy#68] [General]
I notice that the AI response (groq/MAX_TOKENS = 8000) is always short, like a line or two at most. How can I make it longer?

**Comment by Jobus0** [comment]
AlwaysReddy uses a system prompt (look at system_prompts\default_prompt.py) that specifically tells the AI to keep responses that short. I recommend that you make a custom system prompt by copying the default one and editing it to remove those specific instructions.

See these 4-step instructions on how to make a custom system prompt:
https://github.com/ILikeAI/AlwaysReddy?tab=readme-ov-file#how-to-make-a-custom-system-prompt

**Comment by emimix** [comment]
Very useful...thank you.
